backend/service/k8s/job_test.go

```diff
@@ -54,6 +54,24 @@ func testJobClientset() k8s.Interface {
 	return fake.NewSimpleClientset(job)
 }
 
+func TestDescribeJob(t *testing.T) {
+	cs := testJobClientset()
+
+	s := &svc{
+		manager: &managerImpl{
+			clientsets: map[string]*ctxClientsetImpl{"testing-clientset": {
+				Interface: cs,
+				namespace: "testing-namespace",
+				cluster:   "testing-cluster",
+			}},
+		},
+	}
```

Comment:
looks like we had previously been duplicating the same thing [here](https://github.com/lyft/clutch/blob/main/backend/service/k8s/job_test.go#L109-L119) and [here](https://github.com/lyft/clutch/blob/main/backend/service/k8s/job_test.go#L130-L140). can  a helper be created like this [one](https://github.com/lyft/clutch/blob/main/backend/service/k8s/cronjob_test.go#L17-L37) and reused across the delete/create job tests?

Reply:
Made the suggested change.